contracts/SLARegistry.sol

```diff
@@ -104,7 +104,7 @@ contract SLARegistry is ISLARegistry {
             uint128(SLAs.length),
             ipfsHash_,
             extraData_,
-            leverage_
+            governance_
```

Comment:
Not necessary. `cap` is not an argument, nor applies at the SLA level.
It's a simple function of time (SLA periods/epochs).